infra/project-config/outputs.tf

```diff
@@ -34,3 +34,7 @@ output "default_tags" {
 output "github_actions_role_name" {
   value = local.github_actions_role_name
 }
+
+output "aws_ssm_name" {
```

Comment:
is this needed?

Reply:
I actually removed that, but forgot to push last commit. updated now.